integration-test/start_solaris_test.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/docker/containerd/api/grpc/types"
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/go-check/check"
+	"google.golang.org/grpc"
+)
+
+func (cs *ContainerdSuite) TestStartBusyboxLsSlash(t *check.C) {
+	expectedOutput := `bin
+dev
+etc
+export
+home
+lib
+mnt
+opt
+proc
+root
+sbin
+system
+tmp
+usr
+var
+`
+	if err := CreateSolarisBundle("busybox-ls-slash", []string{"ls", "/"}); err != nil {
```

Comment:
I would rather keep this separate initially. While the changes we're proposing are stable a bunch of functionality is currently stubbed (stats, container restart and exit code handling) which will be added going forward. After these are integrated would be a better time to merge test files IMO.
I would be happy to sync the test files when required.